src/discussions/posts/post/Post.jsx

```diff
@@ -86,31 +86,54 @@ function PostHeader({
           </span>
         </div>
       </div>
-      <div className="d-flex mr-3">
-        <Icon src={QuestionAnswer} />
-        <span style={{ minWidth: '2rem' }}>
-          {post.commentCount}
-        </span>
-      </div>
+      {preview
+        ? (
+          <div className="d-flex">
+            <Icon src={QuestionAnswer} />
+            <span style={{ minWidth: '2rem' }}>
+              {post.commentCount}
+            </span>
+          </div>
+        )
+        : <ActionsDropdown commentOrPost={post} actionHandlers={actionHandlers} />}
```

Comment:
The dropdown opens out of visible UI and comes into view after scrolling right. check out here. I think it should open within the original post area.

https://user-images.githubusercontent.com/67791278/133482635-4e93b327-2b9f-4ea2-98b3-62e989b5022c.mov